Cargo.toml

```diff
@@ -324,7 +324,7 @@ postgres-openssl = { version = "0.5.0", default-features = false, features = ["r
 pulsar = { version = "6.3.0", default-features = false, features = ["tokio-runtime", "auth-oauth2", "flate2", "lz4", "snap", "zstd"], optional = true }
 rand = { version = "0.8.5", default-features = false, features = ["small_rng"] }
 rand_distr = { version = "0.4.3", default-features = false }
-rdkafka = { version = "0.35.0", default-features = false, features = ["tokio", "libz", "ssl", "zstd"], optional = true }
+rdkafka = { version = "0.35.0", default-features = false, features = ["cmake-build", "curl", "tokio", "libz", "ssl", "zstd"], optional = true }
```

Comment:
Did you find `cmake-build` was necessary too? I have a vague memory that we were trying to avoid that feature for some reason, but I can't remember why.

Reply:
When I did the initial build, it wouldn't work without the cmake-build. However, since removing it and doing rebuilds with docker it is now building. I'll remove it deeming it a problem with my build environment. :)